Test/Unit/Block/Customer/CreditCardTest.php

```diff
@@ -72,6 +76,10 @@ protected function setUp()
     private function initRequiredMocks()
     {
         $this->contextMock = $this->createMock(Context::class);
+        $this->formKeyMock = $this->getMockBuilder(FormKey::class)
+            ->disableOriginalConstructor()
+            ->setMethods(['getFormKey'])
+            ->getMock();;
```

Comment:
(minor) double semicolon

Reply:
@oxigenka I updated the code. Can you re-check?